### PR TITLE
ec2instanceconnectcli 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by philyoon**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/ec2instanceconnectcli.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
+class Ec2instanceconnectcli < Formula
+  include Language::Python::Virtualenv
+
+  desc "AWS EC2 Instance Connect CLI"
+  homepage "https://aws.amazon.com/cli/"
```
Comment (fxcoudert) [inline Formula/ec2instanceconnectcli.rb]: On the homepage, ec2instanceconnectcli is not mentioned anywhere…

Comment (philyoon) [reply Formula/ec2instanceconnectcli.rb]: Changed to https://github.com/aws/aws-ec2-instance-connect-cli

**Comment by philyoon** [comment]
Huh, didn't know there was a notability requirement. I think a young product like this from a prestigious vendor (AWS in this case) should be an exception from the requirement, but that might be just me.

**Comment by SMillerDev** [comment]
Those projects usually get to the requirements pretty quickly so you can just resubmit when it does.

**Pull request closed without merging**